### PR TITLE
add katex functionality

### PR DESCRIPTION
**Pull request by Jamiller137**
No prerendering yet since that requires nodejs. I mainly just followed their documentation.

closes: #19

### CODE REVIEW

File: examples/custom_clusterer.py
Comment (Jamiller137) [inline examples/custom_clusterer.py:1]: I'm not sure why this is tagged as a difference.

Comment (erooke) [reply examples/custom_clusterer.py:1]: There was a newline character at the bottom of the file which was deleted

**Review by erooke — CHANGES_REQUESTED** [review]
Generally looks good, some small changes and it should be ready to merge

**Comment by erooke** [comment]
Thanks for the contribution! In future if you could avoid merging main into feature branches that makes merging a lot simpler. If you want to update your branch to follow main `git rebase origin/main` will result in a much simpler merge procedure later.

**Comment by Jamiller137** [comment]
Oh I see what you mean on the network graph. Sounds good, thanks!

-Jacob

**Pull request closed without merging**